src/scenes/CreateCollection/CreateCollection.tsx

```diff
@@ -0,0 +1,28 @@
+import { memo } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Wizard, Steps, Step } from 'react-albus';
+import WizardFooter from './WizardFooter';
+import FirstCollection from './steps/FirstCollection';
+import AddNfts from './steps/AddNfts';
+import OrganizeCollections from './steps/OrganizeCollections';
+
+function CreateCollection(_: RouteComponentProps) {
+  return (
+    <Wizard
+      render={({ step, next, previous }) => {
+        return (
+          <>
+            <Steps>
+              <Step id="create" render={FirstCollection} />
```

Comment:
maybe `FirstCollection` can be named `CreateFirstCollection` so all the steps in the wizard have a verb and it's explicit what the intended action is

Reply:
great idea on naming convention- will make the fix and merge!